README.md

```diff
@@ -40,7 +41,9 @@ config/
     └── config.h
 ```
 
-> A sample `SimpleERC20` has already been deployed to Sepolia at `0xa46e0A31a1c248160aCBa9dD354c72E52C92c9f2`.
+File `config.h` must contain proper values for `CONFIG_ERC20_PORTAL_ADDRESS`, `CONFIG_ERC20_CONTRACT_ADDRESS`, and `WITHDRAWAL_ADDRESS`.
+
+> A sample `SimpleERC20` has already been deployed to Sepolia at `0xa46e0A31a1c248160aCBa9dD354c72E52C92c9f2` and may be used to set `CONFIG_ERC20_CONTRACT_ADDRESS`.
```

Comment:
If someone was to use the already deployed `SimpleERC20` contract, how would he get tokens for it?

Reply:
Good point. Perhaps we should just get rid of this information.